OWML.Common/Events.cs

```diff
@@ -1,14 +1,32 @@
 ﻿namespace OWML.Common
 {
+    /// <summary>
+    /// The types of events that can be subscribed to.
+    /// </summary>
     public enum Events
     {
+        /// <summary>Called before the Awake() method.</summary>
```

Comment:
Maybe personal preference, but these could be simplified as just "Called before Awake()". Might also make sense to clarify that this is referring to the MonoBehaviour methods. So something like "Called before MonoBehaviour.Awake()". Leave it up to you to decide what you think makes most sense.

Reply:
Done.